python/katana/plugin.py

```diff
@@ -56,13 +56,13 @@ def load_module(self, fullname):
 
         assert fullname in self._insert_table
 
-        # This will get the module from sys.modules if it is already loaded.
-        module = importlib.import_module(self._insert_table[fullname])
+        aliased_name = self._insert_table[fullname]
         if fullname in sys.modules:
```

Comment:
if fullname was in sys.modules we already returned; aliased_name?

Reply:
This code creates an alias of the module named `aliased_name` at the name `fullname`. Does that clear it up? I'm not really sure what you are trying to ask.

Reply:
there is an if statement at the beginning of the method that checks if fullname in sys.modules. When does fullname change between the two if statements?

Reply:
Ah. I see. You are right. It should be `aliased_name` there. I will fix it. I have no idea how this worked then. Hmm. That's worrisome.